ui/components/User.js

```diff
@@ -67,41 +76,28 @@ const User = (props) => {
   };
 
   const handleGetToken = () => {
-    dataFetch(
-      '/api/token',
-      { credentials: 'same-origin' },
-      (data) => {
+    triggerGetToken()
+      .unwrap()
+      .then((data) => {
         exportToJsonFile(data, 'auth.json');
-      },
-      (error) => ({ error }),
-    );
+      });
   };
 
-  useEffect(() => {
-    dataFetch(
-      '/api/user',
-      {
-        credentials: 'same-origin',
-      },
-      (userData) => {
-        setUser(userData);
-        dispatch(updateUser({ user: userData }));
-      },
-      (error) => ({
-        error,
-      }),
-    );
-  }, [dispatch]);
-
-  useEffect(() => {
-    const { capabilitiesRegistry } = props;
-    if (!capabilitiesLoaded && capabilitiesRegistry) {
-      setCapabilitiesLoaded(true); // to prevent re-compute
-      setAccount(
-        ExtensionPointSchemaValidator('account')(capabilitiesRegistry?.extensions?.account),
-      );
-    }
-  }, [capabilitiesRegistry, capabilitiesLoaded]);
+  if (!userLoaded && isGetUserSuccess) {
+    props.updateUser({ user: userData });
+    setUserLoaded(true);
```

Comment:
Can you confirm why the parent component uses this ,  it might make sense to use a direct rtk call there also